packages/ui/src/Link.module.scss

```diff
@@ -1,5 +1,10 @@
 @use '../styles/constants' as c;
 
+button.buttonReset {
```

Comment:
nit: Should we just include it into `@mixin link`? I mean it should not hurt `a`, right?